ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
What I'm actually concerned about here is that there's wasm code with a `\r\n` in it, which I find very concerning. I assumed you put it there because of Windows; if not, then can we just remove the `\r`?